Source/WebKit/PlatformWPE.cmake

```diff
@@ -528,24 +529,9 @@ if (ENABLE_BREAKPAD)
     )
 endif ()
 
-WEBKIT_BUILD_INSPECTOR_GRESOURCES(${WebInspectorUI_DERIVED_SOURCES_DIR})
-list(APPEND WPEWebInspectorResources_DERIVED_SOURCES
-    ${WebInspectorUI_DERIVED_SOURCES_DIR}/InspectorGResourceBundle.c
-)
-
-list(APPEND WPEWebInspectorResources_LIBRARIES
-    ${GLIB_GIO_LIBRARIES}
-)
-
-list(APPEND WPEWebInspectorResources_SYSTEM_INCLUDE_DIRECTORIES
-    ${GLIB_INCLUDE_DIRS}
-)
+WEBKIT_BUILD_INSPECTOR_GRESOURCES(${WebInspectorUI_DERIVED_SOURCES_DIR} "InspectorGResourceBundle.gresource")
```

Comment:
I think the name of the file could be just `inspector.gresource`, the longer name does not add any additional meaning 😉